src/common/Grid.tsx

```diff
@@ -16,6 +17,7 @@ export function Row(props: RowProps) {
         <div
             className={ "Row" + (props.className !== undefined ? " " + props.className : "") }
             style={ props.style }
+            data-testid = { props.dataTestId }
```

Comment:
The use of test IDs is generally not considered as a good practice (for instance, see [here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text)).

Reply:
I have removed the usage of test id, but I have never able to convert the test using something else. So I just removed the test too.